.github/workflows/docker.yaml

```diff
@@ -0,0 +1,55 @@
+name: Build Docker
+
+on:
+  push:
+    tags: ["v*"]
+  pull_request:
+    paths:
+      - Dockerfile
+      - docker-bake.hcl
+      - ".github/workflows/docker.yaml"
+  schedule:
+    - cron: "44 4 * * *"
+  workflow_dispatch:
+
+jobs:
+  docker:
+    name: Build Docker
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        postgres: ["15", "16", "17"]
+        platform: ["linux/amd64", "linux/arm64"]
+
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: pgduckdb
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Checkout pg_duckdb extension code
+        uses: actions/checkout@v4
+        with:
+          submodules: "recursive"
+      - name: Set env
+        run: |
+          # find the first tag for this commit starting with 'v'
+          echo "POSTGRES_VERSION=${{ matrix.postgres }}" >> $GITHUB_ENV
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          platforms: ${{ matrix.platform }}
+      - name: docker bake
+        uses: docker/bake-action@v5
+        with:
+          targets: pg_duckdb_${{ matrix.postgres }}
+          # push: ${{ github.event_name != 'pull_request' }}
```

Comment:
Let's remove?

Reply:
I was going to put this back in, but, yeah, it's fine.